configuration/docker-compose.yml

```diff
@@ -14,7 +14,14 @@ services:
       - mysql_root_password
     volumes:
       - ./mysql/localhost.sql:/docker-entrypoint-initdb.d/localhost.sql
-      - ./mysql/port_drayage.sql:/docker-entrypoint-initdb.d/port_drayage.sql
+      - ./mysql/localhost.sql:/docker-entrypoint-initdb.d/port_drayage.sql
```

Comment:
@willjohnsonk  is this incorrect. Shouldn't this say ` -./mysql/port_drayage.sql:...` You don't want to map the localhost.sql to the port drayage.sql file since the localhost.sql does not contain information for the Port Drayage database.

Reply:
@paulbourelly999 yea I have no idea how this happened tbh but should be corrected now